buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -101,4 +101,5 @@ if [ "$RUN_TYPEGUARD" == "yes" ]; then $CONDA_INSTALL conda-forge::typeguard; fi
 # echo "DEBUG ENV:"
 # echo "-------------------------------------------------------------------------"
 # conda env export
+conda list --export --explicit
```

Comment:
Can you comment in the `echo`s also please, it's hard to find otherwise.

Reply:
OK, hang on, there is a `set -x`(?) somewhere, because the log lists:

```

https://repo.anaconda.com/pkgs/main/linux-64/certifi-2021.5.30-py37h06a4308_0.conda
https://repo.anaconda.com/pkgs/main/noarch/decorator-5.0.9-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/noarch/ipython_genutils-0.2.0-pyhd3eb1b0_1.conda
https://conda.anaconda.org/numba/label/dev/linux-64/llvmlite-0.38.0dev0-py37hf484d3e_0.tar.bz2
https://repo.anaconda.com/pkgs/main/linux-64/markupsafe-2.0.1-py37h27cfd23_0.conda
https://repo.anaconda.com/pkgs/r/linux-64/numpy-base-1.17.0-py37h2f8d375_0.tar.bz2
https://repo.anaconda.com/pkgs/main/noarch/parso-0.8.2-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/noarch/pickleshare-0.7.5-pyhd3eb1b0_1003.conda
https://repo.anaconda.com/pkgs/main/noarch/ptyprocess-0.7.0-pyhd3eb1b0_2.conda
https://repo.anaconda.com/pkgs/main/noarch/pycparser-2.20-py_2.conda
https://repo.anaconda.com/pkgs/main/linux-64/pyyaml-5.4.1-py37h27cfd23_1.conda
https://repo.anaconda.com/pkgs/main/noarch/smmap-4.0.0-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/noarch/typing_extensions-3.10.0.0-pyh06a4308_0.conda
https://repo.anaconda.com/pkgs/main/noarch/wcwidth-0.2.5-py_0.conda
https://repo.anaconda.com/pkgs/main/noarch/wheel-0.36.2-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/linux-64/cffi-1.14.6-py37h400218f_0.conda
https://repo.anaconda.com/pkgs/main/noarch/gitdb-4.0.7-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/linux-64/jedi-0.17.0-py37_0.conda
https://repo.anaconda.com/pkgs/r/linux-64/numpy-1.17.0-py37h99e49ec_0.tar.bz2
https://repo.anaconda.com/pkgs/main/noarch/pexpect-4.8.0-pyhd3eb1b0_3.conda
https://repo.anaconda.com/pkgs/main/noarch/prompt-toolkit-3.0.17-pyh06a4308_0.conda
https://repo.anaconda.com/pkgs/main/linux-64/setuptools-52.0.0-py37h06a4308_0.conda
https://repo.anaconda.com/pkgs/main/noarch/traitlets-5.0.5-pyhd3eb1b0_0.tar.bz2
https://repo.anaconda.com/pkgs/main/noarch/typing-extensions-3.10.0.0-hd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/noarch/gitpython-3.1.18-pyhd3eb1b0_1.conda
https://repo.anaconda.com/pkgs/main/noarch/jinja2-3.0.1-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/linux-64/pip-21.1.3-py37h06a4308_0.conda
https://repo.anaconda.com/pkgs/main/noarch/pygments-2.9.0-pyhd3eb1b0_0.conda
https://repo.anaconda.com/pkgs/main/linux-64/scipy-1.6.2-py37hf56f3a7_1.conda
https://repo.anaconda.com/pkgs/main/linux-64/ipython-7.22.0-py37hb070fc8_0.conda
# echo "-------------------------------------------------------------------------"
Finishing: Before Install
```

Reply:
I was no longer able to copy and paste from Azure, i think the service is degraded. So here is a screenshot instead:

<img width="682" alt="Screen Shot 2021-07-22 at 12 12 58" src="https://user-images.githubusercontent.com/41563/126623767-8f659f0d-3694-4811-82d7-adf2e07c86c2.png">

Reply:
So yeah, I dunno, it doesn't really make any difference if you comment them in or not?